fireant/database/base.py

```diff
@@ -59,3 +60,19 @@ def fetch(self, query):
             cursor = connection.cursor()
             cursor.execute(query)
             return cursor.fetchall()
+
+    @staticmethod
+    def export_csv(connection, query):
+        """
```

Comment:
Do we also need to do this for Redshift?

Reply:
@mikeengland redshift and snowflake aren't a priority for this feature. we can definitely revisit it though